OrbitVulkanLayer/LayerLogic.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitVulkanLayer/LayerLogic.h"
+
+#include <OrbitBase/Logging.h>
+
+namespace orbit::layer {
+
+VkResult LayerLogic::PreCallAndCallCreateInstance(const VkInstanceCreateInfo* create_info,
```

Comment:
Most comments in this method should be removed. Generally, comments should explain "why" something is done, not "what" (unless it's really hard to understand otherwise). 

I think what would be good here would be to point to layer documentation that is the basis for the implementation here.

Reply:
Removed the ones not explaining a why